lib/entities/npcs/animals/piggy.dart

```diff
@@ -124,7 +124,10 @@ class Piggy extends NPC {
 		StatManager.add(email, Stat.piggies_nibbled);
 		SkillManager.learn(SKILL, email);
 		nibbleCounts[email] = (nibbleCounts[email] ?? 0) + 1;
-
+		messageBus.publish(new RequirementProgress('piggyNibble', email));
+		QuestEndpoint.questLogCache[email].offerQuest('Q11');
+		//Piggy Nibbler Quest
```

Comment:
FYI: generally the comment will go before the line that it is referring to. I'm going to go ahead and pull this anyway so that we can test out the changes.